iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/model/ProjectOrganization.java

```diff
@@ -0,0 +1,54 @@
+package eu.dnetlib.iis.wf.affmatching.model;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.google.common.base.Preconditions;
+
+/**
+ * Paired project and organization identifiers.
+ * 
+ * @author mhorst
+ *
+ */
+public class ProjectOrganization {
```

Comment:
If we change the ProjectToOrganization in the importer to ProjectOrganization then we'll have two classes with the same name. The same will be true in the case of DocumentProject. Think about the DocumentMetadata class, we have 6 classes name like this in different packages of IIS.
In general it is good to have different names for classes if possible.